analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -702,6 +712,8 @@ class Pip(
         getLicenseFromLicenseField(map["License"]?.single())?.let { declaredLicenses += it }
         map["Classifiers"]?.mapNotNullTo(declaredLicenses) { getLicenseFromClassifier(it) }
 
+        val authors = map["Author"]?.singleOrNull()?.takeIf(::isValidAuthor)?.let { sortedSetOf(it) } ?: sortedSetOf()
```

Comment:
Just wondering, do you think it makes sense to generalize `parseAuthors` to take a nullable string so it could also be used here?

Reply:
I extracted a `parseAuthorString()` function that expects a nullable string. It is now used from both places.